sys-fs/tmsu/tmsu-0.6.1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit golang-build
+
+EGO_PN="github.com/oniony/TMSU/"
+DESCRIPTION="Files tagger and virtual tag-based filesystem"
+HOMEPAGE="https://github.com/oniony/TMSU/wiki"
+SRC_URI="https://github.com/oniony/TMSU/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+
+RDEPEND="
+	dev-go/go-sqlite3
+	dev-libs/go-fuse
```

Comment:
I'm not a go expert. I've checked the link-time dependencies for tmsu:
```
$ scanelf -n tmsu 
 TYPE   NEEDED FILE 
ET_EXEC libdl.so.2,libpthread.so.0,libc.so.6 tmsu
```

It doesn't seem to dynamically link against any of these packages. Would these still be required at runtime?